code/datums/antagonists/datum_changeling.dm

```diff
@@ -0,0 +1,60 @@
+/datum/antagonist/changeling
+	name = ROLE_CHANGELING
+
+	text_on_lose = "You are no longer a changeling!"
+
+	give_special_equipment = FALSE
+
+	number_of_possible_objectives = 2
+	possible_objectives = list(/datum/objective/steal, /datum/objective/assassinate, /datum/objective/maroon)
+
+/datum/antagonist/changeling/apply_innate_effects()
+	if(!owner)
+		return
+	if(owner.current)
+		owner.current.make_changeling()
+
+	ticker.mode.update_changeling_icons_added(owner)
```

Comment:
Honestly, antag huds need a complete overhaul.

I wanted there to be a hud for people who sold their soul, however, that would overwrite the previous antag hud, so fellow gang members would see the no-soul aura, but not see the gang hud.